GooglePlacesAutocomplete.js

```diff
@@ -799,6 +800,7 @@ GooglePlacesAutocomplete.defaultProps = {
   textInputProps: {},
   enablePoweredByContainer: true,
   predefinedPlaces: [],
+  getCurrentLocation: navigator.geolocation.getCurrentPosition,
```

Comment:
`navigator` is deprecated in the latest version of React Native.  This wouldn't work past React Native v0.60.0